File: verify/except/loop_try.py
def verify(*args):
    res = ""
    for i in range(12):
        res += "i = {}\n".format(i)
        if i == 3:
            res += "Found 3\n"
        if i != 7:
            res += "Always except 7 ({}).\n".format(i)
    return (0, res + "Loop exited at 11.\n", "")

data = [
    [],
]

File: failures/loop_try.c
int foo() {
    int i;
    for (i = 0; 1; i++) {
        try {
            printf("i = %d\n", i);
            switch (i) {
                case 7: continue; //!
                case 3: throw Three(NULL);
                case 11: throw Eleven(NULL);
            }
        } catch (Three _) {
            printf("Found 3\n");
        } finally {
            printf("Always (%d).\n", i);
        }
    }
    printf("Never.\n");
}

int main() {
    try {
        foo();
    } catch (Three _) {
        printf("Never.\n");
    } catch (Eleven _) {
        printf("Loop exited at 11.\n");
    }
}
